plugins/irma-web/index.js

```diff
@@ -15,14 +15,26 @@ module.exports = class IrmaWeb {
         // Check for validity of function to prevent errors when multiple events are cached.
         if (this._stateMachine.isValidTransition(t))
           this._stateMachine.transition(t, this._lastPayload);
+      },
+      (pairingCode) => {
+        console.log(pairingCode, this._lastPayload);
```

Comment:
Note to self: remove debug logging (or do this conditionally when `debugging` is set to `true`